hetzner/README-cloud.md

```diff
@@ -1,18 +1,23 @@
 # Table of Contents
 
-   * [Introduction](#introduction)
-      * [Scope](#scope)
-   * [Requirements](#requirements)
-      * [Install hcloud CLI](#install-hcloud-cli)
-   * [Hetzner Cloud](#hetzner-cloud)
-      * [Init the hetzner context](#init-the-hetzner-context)
-      * [Create the hetzner SSH key](#create-the-hetzner-ssh-key)
-   * [The inventory](#the-inventory)
-   * [The VM](#the-vm)
-      * [Create the hetzner vm](#create-the-hetzner-vm)
-      * [Delete a hetzner server](#delete-a-hetzner-server)
+* [Table of Contents](#table-of-contents)
+* [Introduction](#introduction)
+   * [Scope](#scope)
+* [Requirements](#requirements)
+   * [Install hcloud CLI](#install-hcloud-cli)
+* [Hetzner Cloud](#hetzner-cloud)
+   * [Init the hetzner context](#init-the-hetzner-context)
+* [The inventory](#the-inventory)
+* [The VM](#the-vm)
+   * [All-in-one creation](#all-in-one-creation)
+   * [Create the hetzner SSH key](#create-the-hetzner-ssh-key)
+   * [Create the hetzner vm](#create-the-hetzner-vm)
+   * [Delete a hetzner server](#delete-a-hetzner-server)
    * [Next steps](#next-steps)
-      * [Steps to create a k8s / okd cluster](#steps-to-create-a-k8s--okd-cluster)
+* [Steps to create a k8s / okd cluster](#steps-to-create-a-k8s--okd-cluster)
```

Comment:
As you removed `okd` from the title then this TOC line should be updated ;-)

Reply:
fixed